cmd/zed/compact/command.go

```diff
@@ -37,6 +38,9 @@ func (c *Command) Run(args []string) error {
 	if err != nil {
 		return err
 	}
+	if _, err := rlimit.RaiseOpenFilesLimit(); err != nil {
+		return err
+	}
```

Comment:
@nwt wanted to get your opinion on this. Should this go in lake/api/local since this wouldn't effect remote clients?